packages/libs/client-utils/src/marmalade/config.ts

```diff
@@ -0,0 +1,335 @@
+import type { BuiltInPredicate } from '@kadena/client';
+import type { IPactDecimal, IPactInt } from '@kadena/types';
+
+/** -----------COMMON----------- */
+
+export interface CommonProps {
+  meta?: {
+    senderAccount?: string;
+    gasLimit?: number;
+    gasPrice?: number;
+  };
+  capabilities?: {
+    name: string;
+    props: any[];
+  }[];
+  additionalSigners?: {
+    keyset: {
+      keys: string[];
+      pred: BuiltInPredicate;
+    };
+    capabilities: {
+      name: string;
+      props: any[];
+    }[];
+  }[];
+}
+
+/** -----------COMMON----------- */
+
+/** -----------CREATE----------- */
+
+export interface IRoyaltyInfoInput {
+  fungible: {
+    refName: {
+      name: string;
+      namespace: string | null;
+    };
+    refSpec: [
+      {
+        name: string;
+        namespace: string | null;
+      },
+    ];
+  };
+  creator: {
+    account: string;
+    keyset: {
+      keys: string[];
+      pred: BuiltInPredicate;
+    };
+  };
+  royaltyRate: IPactDecimal;
+}
+
+export interface IGuardInfoInput {
+  mintGuard?: {
+    keys: string[];
+    pred: BuiltInPredicate;
+  };
+  uriGuard?: {
+    keys: string[];
+    pred: BuiltInPredicate;
+  };
+  saleGuard?: {
+    keys: string[];
+    pred: BuiltInPredicate;
+  };
+  burnGuard?: {
+    keys: string[];
+    pred: BuiltInPredicate;
+  };
+  transferGuard?: {
+    keys: string[];
+    pred: BuiltInPredicate;
+  };
+}
+
+export interface ICollectionInfoInput {
+  collectionId: string;
+}
+
+export interface ICreateTokenPolicyConfig {
+  customPolicies?: boolean;
+  upgradeableURI?: boolean;
```

Comment:
Our function is named as `update-uri`, and we've named the interface that contains the update-uri functions as `kip.updatable-uri-policy-v1`, so I suggest using `updatableURI` in this tool, as well.